Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/RiskLevelChangeDetector.kt

```diff
@@ -99,14 +99,16 @@ class RiskLevelChangeDetector @Inject constructor(
 
         // Save most recent date of high or low risks
         if (newRiskState.riskState in listOf(RiskState.INCREASED_RISK, RiskState.LOW_RISK)) {
+            Timber.d("newRiskState=$newRiskState")
+            val lastRiskEncounterAt = newRiskState.lastRiskEncounterAt ?: newRiskState.calculatedAt
```

Comment:
@d4rken  I noticed that `lastRiskEncounterAt` after a fresh install is null. while Home Screen show Low risk for example. as a fallback  `calculatedAt` is used

Reply:
Clarified it with @d4rken  and @mlenkeit , Test Donor will provide a default value  (`-1`) if the date is missing